site/docusaurus.config.js

```diff
@@ -0,0 +1,201 @@
+        textColor: '#fff',
+        backgroundColor: '#3480eb',
+      },
+      navbar: {
+        title: 'Phylum Docs',
+        logo: {
+          alt: 'Phylum Logo',
+          src: 'img/phylum_logo.svg',
+        },
+        hideOnScroll: false,
+        items: [
+          {
+            type: 'docSidebar',
+            sidebarId: 'docsSidebar',
+            position: 'left',
+            label: 'Docs',
+          },
+          {
+            href: 'https://api.phylum.io/api/v0/swagger/index.html',
+            position: 'left',
+            label: 'API',
+          },
+          {
+            href: 'https://github.com/phylum-dev',
+            position: 'right',
+            className: 'header-github-link',
+            'aria-label': 'GitHub repository',
+          },
+        ],
+      },
+      footer: {
+        style: 'dark',
+        copyright: `Copyright © 2020-${new Date().getFullYear()} Phylum, Inc.`,
+        links: [
+          {
+            label: 'Phylum Home',
+            href: 'https://phylum.io',
+          },
+          {
+            label: 'Discord',
+            href: 'https://discord.gg/c9QnknWxm3',
+          },
+          {
+            label: 'Twitter',
+            href: 'https://twitter.com/Phylum_IO',
+          },
+          {
+            label: 'GitHub',
+            href: 'https://github.com/phylum-dev',
+          },
+          {
+            label: 'YouTube',
+            href: 'https://www.youtube.com/@phylum_io',
+          },
+          {
+            label: 'LinkedIn',
+            href: 'https://www.linkedin.com/company/phylum-io',
+          },
+          {
+            label: 'DEV',
+            href: 'https://dev.to/phylum',
+          },
+        ],
+      },
+      prism: {
+        theme: prismThemes.github,
+        darkTheme: prismThemes.dracula,
+        // The default languages for code block highlighting are found here:
+        // https://github.com/FormidableLabs/prism-react-renderer/blob/master/packages/generate-prism-languages/index.ts#L9
+        // Choose additional languages from here: https://prismjs.com/#supported-languages
+        // When there are aliases, use the name of the first one in the list containing the entry you actually use.
+        additionalLanguages: [
+          'bash',
+          'javascript',
+          'json',
+          'python',
+          'rego',
+          'shell-session',
+          'toml',
+        ],
+      },
+      docs: {
+        sidebar: {
+          hideable: true,
+          autoCollapseCategories: true,
+        }
+      },
+      tableOfContents: {
+        minHeadingLevel: 2,
+        maxHeadingLevel: 4,
+      },
+    }),
+
+  markdown: {
+    format: 'detect',
+    mermaid: true,
+    // TODO: Progressively turn off MDXv1 compatibility
+    //       https://docusaurus.io/docs/migration/v3#turn-off-mdx-v1-compat
+    mdx1Compat: {
+      comments: true,
+      admonitions: true,
+      headingIds: true,
```

Comment:
Could probably go ahead and turn these off. I don't think we use admonitions of heading IDs anywhere currently.